.github/workflows/e2e.yml

```diff
@@ -16,9 +16,10 @@ jobs:
       # the gh tag of system-test repo version to run
       SYSTEM_TEST_GIT_REF: master
 
-      # the soroban tools source code to compile and run from system test
+      # the soroban CLI & RPC source code to compile and run from system test
       # refers to checked out source of current git hub ref context
-      SYSTEM_TEST_SOROBAN_TOOLS_REF: ${{ github.workspace }}/soroban-tools
+      SYSTEM_TEST_SOROBAN_CLI_REF: https://github.com/stellar/soroban-tools.git\#main
+      SYSTEM_TEST_SOROBAN_RPC_REF: https://github.com/stellar/soroban-rpc.git\#main
```

Comment:
you want to build the rpc code from this repo correct? this path should point to gha workspace which has the branch code checked out locally:
```
${{ github.workspace }}/soroban-rpc
```

Reply:
Yes, I would like to build from this repo. Strangely enough, `${{ github.workspace }}/soroban-rpc` produce the same error discussed in [this thread](https://stellarfoundation.slack.com/archives/G01FUHTUQ8Z/p1706656493328379). Looking into this more

Reply:
The issue was not adding a checkout action for `soroban-rpc`